<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>180</y>
     <width>71</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="UserNameLabel">
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>用户名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="UserPasswordLabel">
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>密 码</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>180</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="UserNameInput"/>
    </item>
    <item>
     <widget class="QLineEdit" name="UserPasswordInput"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>440</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="RegisterButton">
      <property name="text">
       <string>注册</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="DetailButton">
      <property name="text">
       <string>详情</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="LoginButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>300</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/new/login/photos/button.png);
color:white;
boder:5px;
border-radius: 1px solid rgb(25, 65, 91);</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="loginPic">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>0</y>
     <width>611</width>
     <height>531</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="loginPic.qrc">:/new/login/photos/login.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="LoginLabel">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>90</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bauhaus 93</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(33, 84, 118)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>LogIN</string>
   </property>
  </widget>
  <widget class="QLabel" name="StockStimulatorLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>561</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Forte</family>
     <pointsize>48</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>StockStimulator</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginCloseButton">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>0</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>500</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>虚拟股票交易系统</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginMinimumButton">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>0</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent</string>
   </property>
   <property name="text">
    <string>—</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="loginPic.qrc"/>
 </resources>
 <connections/>
</ui>
